### ISSUE TITLE
Table order in `basic-cmip6-file_def_nemo.xml`

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#808]
Preserving the earlier (accidental python2) ordering in the `basic-cmip6-file_def_nemo.xml` of the file definitions in its production chain. In commit 76a40e7 (related to the python3 porting #403) this preserved ordering was enforced to large extent, however the lacking part is the table ordering within a file definition part (this order still appeared not to be guaranteed; until now this did not manifestate). The order vulnerability can be triggered by a shift in version of `python` and/or `xml.etree.ElementTree` but also by the `drq` `.xlsx` ordering or the `checkvars` `.xlsx` writing. It can have multiple sources however the consequence is rather inconvenient: when the order in the `basic-cmip6-file_def_nemo.xml`  changes the order in all the (thousands) produced `output-control-files` change which are stored under the ECE3 svn repository - this we want to avoid because it would cause updates of all those files and the related validation work.

Therefore we want a preserved table order for the `basic-cmip6-file_def_nemo.xml` for those parts of the file where this applies. The variable order is until now not an issue, so we leave that as long it is not necessary as it would be lot of admin (fortunately this sub order is until now alphabetic). Note that for the solution in mind the  `basic-cmip6-file_def_nemo.xml` order is conserved but the order in the `basic-flat-cmip6-file_def_nemo.xml` is not, however the content of the flat one can be only identical (appart of the order) when the `basic-flat-cmip6-file_def_nemo.xml` is identical, which makes validation easy and this only concerns the `genecec` side of the story.

**Comment by treerink** [comment]
Solved in 1e034602887e33baee1adb5c60e32de117782aa1.

**Comment by treerink** [comment]
`genecec` has been tested and its results remain now identical after reapplying:
```
cd ${HOME}/cmorize/ece2cmor3/ece2cmor3/scripts/create-nemo-only-list/
./create-nemo-only-list.sh
cd ../
./create-basic-ec-earth-cmip6-nemo-namelist.py config-create-basic-ec-earth-cmip6-nemo-namelist
```
And running `genecec` thereafter:
```
./run-genecec.sh default 443 ece2cmor3
./run-genecec.sh pextra  443 ece2cmor3
```
